step04-animation/build.gradle

```diff
@@ -44,8 +44,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.1.0-alpha01'
     implementation 'androidx.recyclerview:recyclerview:1.1.0-alpha01'
-    //    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
-    implementation files('libs/constraint-debug.aar')
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha3'
```

Comment:
Do you need the alpha build? I think you said you did but I wanted to makes sure.

Reply:
Correct, MotionLayout is coming and that will be a nice chunky delete delete delete PR :)